app/api/events.py

```diff
@@ -64,23 +111,30 @@ class Meta:
                            many=True,
                            type_='sponsor')
 
+    user = Relationship(attribute='user',
+                        self_view='v1.event_user',
+                        self_view_kwargs={'id': '<id>'},
+                        related_view='v1.user_detail',
+                        related_view_kwargs={'id': '<id>'},
+                        schema='UserSchema',
+                        type_='user')
+
 
 class EventList(ResourceList):
-    decorators = (jwt_required, )
-    schema = EventSchema
-    data_layer = {'session': db.session,
-                  'model': Event}
 
+    def before_create_object(self, data, view_kwargs):
+        data['starts_at'] = data['starts_at'].replace(tzinfo=None)
+        data['ends_at'] = data['ends_at'].replace(tzinfo=None)
```

Comment:
@magdalenesuo 
I think it is due to this [pull # 3580](https://github.com/fossasia/open-event-orga-server/pull/3580)
**Made Event and Session API responses timezone aware in ISO8601 format**